libvast/src/system/evaluator.cpp

```diff
@@ -134,9 +134,7 @@ void evaluator_state::decrement_pending() {
   if (--pending_responses == 0) {
     // Now we ask the store for the actual data.
     // TODO: handle count estimate requests.
-    self->send(store, atom::exporter_v, caf::actor_cast<caf::actor>(client));
-    promise.delegate(store, atom::extract_v, hits,
-                     static_cast<receiver<table_slice>>(client));
+    promise.deliver(hits);
```

Comment:
rename `decrement_pending` to something more fitting.

Reply:
After thinking about it I am of the opinion that `decrement_pending` isn't such a bad name for this function.

Reply:
Alrighty. I have no hard feelings on this.